pages/home.html

```diff
@@ -286,132 +264,118 @@ <h1 class="text-center py-8">Features</h1>
       </div>
     </section>
 
-    <section class="px-20 py-20">
-      <h1 class="text-center leading-tight">
+    <section class="customer">
+      <h1 class="customer__title">
         The stunning results our<br />
         customers have experienced
       </h1>
-      <div class="grid grid-cols-3 gap-8 py-10">
-        <div
-          class="flex flex-col justify-center items-center gap-5 border border-blue-500 p-10 rounded-2xl"
-        >
-          <p class="text-center paragraph-large">
+      <div class="customer__content">
+        <div class="customer__card">
+          <p class="customer__card-text">
             Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
             nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
             erat, sed diam voluptua.
           </p>
-          <div class="flex flex-col justify-center items-center gap-1">
+          <div class="customer__card-image">
             <img src="/assets/images/person1.png" alt="photo-1" />
-            <p class="text-2xl">Ron Wood</p>
-            <p class="text-light text-xs">CEO</p>
+            <p class="customer__card-name">Ron Wood</p>
+            <p class="customer__card-position">CEO</p>
```

Comment:
This name is not goot I suggest you name it like customer__card-center or left anywhere. we need o know where it is positioned